pkg/webhook/workspace/handler/immutable.go

```diff
@@ -17,13 +17,23 @@ import (
 	"reflect"
 
 	"github.com/devfile/devworkspace-operator/pkg/config"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
That changes pretty nothing but out `make fmt` does not seem to sort imports nicely... Like that apimachinery import is separated from other apimachinery imports. If I sort them manually I'd put them like:
```go
	devworkspace "github.com/devfile/kubernetes-api/pkg/apis/workspaces/v1alpha1"

	"github.com/devfile/devworkspace-operator/pkg/config"

	"github.com/google/go-cmp/cmp"
	"github.com/google/go-cmp/cmp/cmpopts"

	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
	"k8s.io/apimachinery/pkg/runtime"
	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
```
^ good thing here that make fmt does not touch such manually sorted list as well )
Just for us an item to take a look later

Reply:
Yeah the issue is that goimports will preserve whitespace dividing sections. The only actual formatting it does is to put libraries at the top, separated from other imports, and sorts imports within each section as usual.

Reply:
Yeah the issue is that goimports will preserve whitespace dividing sections. The only actual formatting it does is to put libraries at the top, separated from other imports, and sorts imports within each section as usual.

Reply:
Fixed